src/components/checkbox/__tests__/__snapshots__/CdrCheckbox.spec.js.snap

```diff
@@ -19,15 +19,15 @@ exports[`CdrCheckbox renders correctly 1`] = `
     <span
```

Comment:
In addition to the snapshot updates, I did run a backstop.js comparison and found no changes.